### PR TITLE
Change Items.Add to Items.Append

### PR DESCRIPTION
**Pull request by nxtn**
[ListView.Items](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.itemscontrol.items#Windows_UI_Xaml_Controls_ItemsControl_Items) is a [ItemCollection](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.itemcollection).

### CODE REVIEW

**Comment by garycentric** [comment]
@jwmsft: The PR **passed** the acceptance checklist. Please evaluate further and approve/complete the PR if appropriate. Thanks!

File: windows-apps-src/design/controls-and-patterns/listview-and-gridview.md [hunk 1]
```diff
@@ -80,11 +80,11 @@ Here's the list view created in code. The resulting list is the same as the one
 ```csharp
 // Create a new ListView and add content. 
 ListView listView1 = new ListView(); 
-listView1.Items.Add("Item 1"); 
-listView1.Items.Add("Item 2"); 
-listView1.Items.Add("Item 3"); 
-listView1.Items.Add("Item 4"); 
-listView1.Items.Add("Item 5");
+listView1.Items.Append("Item 1");
```
Comment (MikeHillberg) [inline windows-apps-src/design/controls-and-patterns/listview-and-gridview.md:83]: Add is correct. 

Append is an extension method from the System.Linq.Enumerable class. It returns a new enumerable with the item appended to the current enumeration, but it doesn't modify the original list (doesn't modify listView1.Items). The purpose of this example is to call ICollection<object>.Add.

**Review by MikeHillberg — CHANGES_REQUESTED** [review]

File: windows-apps-src/design/controls-and-patterns/listview-and-gridview.md (hunk 1 above)
Comment (nxtn) [reply windows-apps-src/design/controls-and-patterns/listview-and-gridview.md:83]: [ItemCollection.Append](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.itemcollection.append#Windows_UI_Xaml_Controls_ItemCollection_Append_System_Object_)

[ItemCollection](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.itemcollection) doesn't implement any interface.

Comment (MikeHillberg) [reply windows-apps-src/design/controls-and-patterns/listview-and-gridview.md:83]: At the WinRT level, ItemCollection implements IVector<IInspectable>, which does have an Append method. But in the projection to C#, IVector<T> projects as IList<T>, which has an Add method instead (from ICollection<T>). So the members of ItemCollection depend on what language you're using.

I think the issue here is that the [ItemCollection](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.itemcollection) pages aren't correct.

Comment (nxtn) [reply windows-apps-src/design/controls-and-patterns/listview-and-gridview.md:83]: Thanks. I was using C++ so I saw the `Append` method instead of `Add`.

**Comment by MikeHillberg** [comment]
I opened a separate issue on ItemCollection class page to update/explain the different language projections.

**Pull request closed without merging**